.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image: circleci/node:10.16
+      - image: circleci/node:14.17
```

Comment:
with a bump in jest version, it caused an issue with [jsdom](https://github.com/jsdom/jsdom/issues/2795) which is required by the a11y tests. Bumping the node version to 14 (which is something that was needed) to see if resolves the circleci build errors.

Reply:
did resolve the issue.